hack/e2e/environment.sh

```diff
@@ -18,8 +18,8 @@ function os_check() {
 }
 
 # metal3-dev-env customization
-export CAPI_VERSION=${CAPI_VERSION:-"v1alpha4"}
-export CAPM3_VERSION=${CAPM3_VERSION:-"v1alpha5"}
+export CAPI_VERSION=${CAPI_VERSION:-"v1beta1"}
+export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export NUM_NODES=${NUM_NODES:-"4"}
```

Comment:
maybe this could be separately?

Reply:
Any reason, we have v1b1 e2e jobs already and they passed, what prevents us from it?

Reply:
just suggesting to do it separately, because form the PR title and description I was not expecting e2e to be changed., But not a blocked, fine for me.

Reply:
Hm, but PR description states changing e2e tests.